src/pages/Content/ControlBar.tsx

```diff
@@ -202,7 +203,7 @@ export default function ControlBar({ onExit }: { onExit: () => void }) {
   const displayedSelector = getBestSelectorForAction(
     {
       type: ActionType.Click,
-      tagName: hoveredElement?.tagName ?? '',
+      tagName: (hoveredElement?.tagName ?? '') as TagName,
```

Comment:
I'm not 100% sure if this is the right way to type tagName as it can very well be not enumerated in TagName, but perhaps this is helpful enough

Reply:
I'll view this as a good reminder that tells we should probably handle unexpected tagName before passing it to getBestSelectorForAction